packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -420,11 +419,8 @@ export default function PostOptionsMenu({
             target_id: TargetId.ContextMenu,
           });
         }
-        router.push(
-          `${webappUrl}feeds/${router.query?.slugOrId || user.id}/edit?dview=${
```

Comment:
Do we go to settings even if its not main feed?

Reply:
For AI settings, it doesn't really matter?

Reply:
It might matter in the future so maybe best to leave it handled.

Reply:
I'll fix it

Reply:
Fixed it